fineract-provider/build.gradle

```diff
@@ -444,13 +442,10 @@ tasks.withType(JavaCompile) {
                 "DoubleBraceInitialization",
                 "InconsistentCapitalization",
                 "MissingOverride",
-//                "ReturnMissingNullable",
```

Comment:
This https://errorprone.info/bugpattern/ReturnMissingNullable could actually be nice, for NPE safety (which is something I was quite passionate about, a while ago, see http://www.lastnpe.org) - but of course depends on the effort... how many violations are there for this?

Reply:
It was removed for version 2.4.0